### DISCUSSION TITLE
Quarry with frame not attached

### DISCUSSION THREAD

**Discussion by Lumengrid** [Rearth/Oritech#216] [Ideas]
Given the Destroyer with the Quarry addon I would like to have the possibility to define the frame limits 
without having it attached to the destroye, in order to not lose the space occupied by the frames 
that I could use for other addons
Ty

![2024-12-22_11 32 16](https://github.com/user-attachments/assets/aa53de1d-23e0-4a7b-96c6-11327471dcd5)

**Comment by Rearth** [comment]
This is not really planned. Not being able to place addons there is a desired and intended limitation.
